frontend/android/build.gradle

```diff
@@ -105,6 +105,8 @@ dependencies {
 
   debugImplementation project(':debug-updater')
 
+  implementation deps.rx.relay
```

Comment:
Rx is intentionally confined inside the search presenter and I don't want to expose it across module boundaries. This isn't the correct way to do it, though, even if we were going to. The dependency would need changed to `api` in the presenter.

The Java version of `Consumer` is used because it was already a public type shared between the two. An easy way to achieve the same thing is to use a Kotlin lambda type `(Event) -> Unit`.

Reply:
@JakeWharton ok, I will revert my Rx changes.